src/entities/Place/routes/index.ts

```diff
@@ -28,4 +29,6 @@ export default routes((router) => {
   router.get("/places/:place_id", getPlace)
   router.get("/places", getPlaceList)
   router.put("/places/:place_id/rating", updateRating)
+  router.get("/places/:place_id/categories", getPlaceCategories)
+  router.patch("/places/:place_id/update_rating", updateRating)
```

Comment:
This is old code, this route is different in prod
```suggestion
  router.put("/places/:place_id/rating", updateRating)
```

Reply:
excellent catch!